app/models/company.rb

```diff
@@ -17,4 +17,5 @@ def d3_coordinates
     slice(:longitude, :latitude)
   end
 
+  STATUS_OPTIONS = { "Uncontacted" => "uncontacted", "Contacted" => "contacted", "Confirmed" => "confirmed", "Dead" => "dead" }
```

Comment:
I guess we don't need another before_save since there's a dropdown which hardcodes the status option as it comes in? Status is always downcased right?